docs/source/reservations/reservations.rst

```diff
@@ -0,0 +1,182 @@
+
+.. _reservation:
+
+Reservations & DWS
+==================
+
+
+Clouds are designed to be elastic, but due to the recent GPU shortages, many organizations have started to get reservations
+from cloud providers to lock in GPU capacity.
+
+SkyPilot can be used to request resources from reservations and even combine them with on-demand/spot resources to fully
+utilize the capacity in your cloud accounts.
+
+AWS capacity reservations & capacity block
+------------------------------------------
+
+AWS capacity reservations and capacity block offers are a way to lock in a certain amount of compute capacity for a period of time. The latter is for high-end GPUs, such as A100s(P4d instances) and H100s(P5d instances), while the former is for all other instance types.
+Instead of committing to a 1-3 year long contract, you can get a capacity reservation and capacity block for as short as 1 second and 1 day, respectively.
```

Comment:
```suggestion
AWS **capacity reservations** and **capacity blocks** are ways to reserve a certain amount of compute capacity for a period of time. The latter is for high-end GPUs, such as A100s (P4d instances) and H100s (P5d instances), while the former is for all other instance types.
Instead of committing to a 1-3 year long contract, you can get a capacity reservation or capacity block for as short as 1 second or 1 day, respectively.
```